### PR TITLE
Added a signed version of v3.

### PR DESCRIPTION
**Pull request by evrii**
Description:
Added a signed version of DUCC v3.
Renders v3 correctly

https://user-images.githubusercontent.com/24514630/169584708-85ec2830-1558-4c94-ab5a-9754efdf7f41.mov

Resizes correctly

https://user-images.githubusercontent.com/24514630/169585018-f97d0966-c8df-4b23-a566-cc5283b202e3.mov

---
**PR checklist**

- [x] This PR meets the Acceptance Criteria in the JIRA story
- [x] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have added explanatory comments where the logic is not obvious
- [ ] I have run and tested this change locally, and my testing process is described here
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later
- [ ] If this includes an API change, I have run the E2E tests on this change against my local server with [yarn test-local](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples) because this PR won't be covered by the CircleCI tests 
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P)
- [ ] If this change impacts deployment safety (e.g. removing/altering APIs which are in use) I have documented these in the description
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and updated the appropriate API consumers
  * AoU UI
  * [Perf tests](https://github.com/broadinstitute/mcnulty/blob/develop/src/test/scala/services/AoU.scala)
  * [Researcher Directory export](https://github.com/all-of-us/workbench/wiki/Researcher-Directory-(RDR-export))
  * Cron tasks - for Offline*Controllers
  * SumoLogic - for EgressAlert

### CODE REVIEW

File: ui/src/app/pages/profile/profile-component.spec.tsx [hunk 1]
```diff
@@ -160,13 +162,4 @@ describe('ProfilePageComponent', () => {
       wrapper.find('[data-test-id="signed-ducc-panel"]').exists()
     ).toBeFalsy();
   });
-
-  it('should not display a link to the signed DUCC if the user has signed an older DUCC', async () => {
-    updateProfile({ duccSignedVersion: 3 });
```
Comment (evrii) [inline ui/src/app/pages/profile/profile-component.spec.tsx:165]: We could either delete this test or change it to an even older version of DUCC (v2?).

File: ui/src/app/components/data-user-code-of-conduct.spec.tsx [hunk 2]
```diff
@@ -186,16 +186,4 @@ describe('DataUserCodeOfConduct', () => {
       wrapper.find('[data-test-id="ducc-signature-page"]').exists()
     ).toBeFalsy();
   });
-
-  it('should not display the Content and Signature pages in SIGNED mode if the user has signed an older DUCC', async () => {
-    updateProfile({ duccSignedVersion: 3 });
```
Comment (jmthibault79) [inline ui/src/app/components/data-user-code-of-conduct.spec.tsx:191]: we can retain this test if we make this v2

File: ui/src/app/pages/profile/profile-component.spec.tsx (hunk 1 above)
Comment (jmthibault79) [reply ui/src/app/pages/profile/profile-component.spec.tsx:165]: yeah I'd like to keep it v2

**Review by jmthibault79 — COMMENTED** [review]
The file looks good (identical w/ old version except whitespace) but the image is not rendering - assuming it needs a path update
Screen Shot 2022-05-20 at 3 21 34 PM

**Comment by evrii** [comment]
> The file looks good (identical w/ old version except whitespace) but the image is not rendering - assuming it needs a path update Screen Shot 2022-05-20 at 3 21 34 PM

Whoops. Thanks for pointing this out. This should be good now.

**Review by jmthibault79 — APPROVED** [review]

**Pull request merged**